modules/ldap_groups/lib/tasks/ldap_groups.rake

```diff
@@ -58,36 +58,6 @@ namespace :ldap_groups do
       ldif = ENV.fetch('LDIF_FILE') { Rails.root.join('spec/fixtures/ldap/users.ldif') }
       ldap_server = Ladle::Server.new(quiet: false, port: '12389', domain: 'dc=example,dc=com', ldif:).start
 
-      puts <<~INFO
```

Comment:
I moved this bit below the job execution as it gets lost in the log output otherwise